service/job/job_type_controller.py

```diff
@@ -242,6 +455,7 @@ def get_job_type_detail(job_type):
 
     :reqheader Accept: application/json
     :param str job_type: Name of the job type
+    :param str schema: (optional) if set to 'true', parameter schema is served
```

Comment:
Hmm, die Funktion hat aber kein Param Schema?

Reply:
Verstehe.. Die IDE meckert sonst halt, weil sie das als Funktionsparameter interpretiert. Vielleicht einfach in den normalen Docstring, wo das nicht durch Dokuprogramme, IDEs etc. ausgewertet und falsch interpretiert wird?

Reply:
Bin auch dafür, dass das irgendwo anders im docstring untergebracht wird.

Reply:
Jo war sowieso fehlerhaft, Ist jetzt geändert, schaut mal ob die IDE damit klarkommt.

Reply:
Super